app/components/Databar.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Button, NumericInput, Popover, H5, H6, Tooltip, Icon } from '@blueprintjs/core';
+import s from './Databar.scss';
+
+type Props = {
+  filePath: string | undefined;
+  onDataImportClick: Function;
+  onEventsUpdateClick: Function;
+  onDataExportClick: Function;
```

Comment:
`onEventsUpdateClick: (eventDuration: number | undefined) => void;`

we specify value in minutes so this would be number or undefined, instead of Duration type from date-fns
fixed